server/base/src/test/java/org/apache/accumulo/server/fs/VolumeManagerImplTest.java

```diff
@@ -105,7 +115,8 @@ public String choose(VolumeChooserEnvironment env, String[] options) {
   @SuppressWarnings("deprecation")
   private static final Property INSTANCE_DFS_URI = Property.INSTANCE_DFS_URI;
 
-  @Test
+  // Expected to throw a runtime exception when the WrongVolumeChooser picks an invalid volume.
+  @Test(expected = RuntimeException.class)
```

Comment:
Using ExpectedException JUnit rule allows more fine checking of the thrown exception.